ui-core/src/main/java/com/adyen/checkout/ui/core/internal/ui/view/SecurityCodeInput.kt

```diff
@@ -15,7 +15,7 @@ class SecurityCodeInput @JvmOverloads constructor(
     context: Context,
     attrs: AttributeSet? = null,
     defStyleAttr: Int = 0
-) : CardNumberInput(context, attrs, defStyleAttr) {
+) : AdyenTextInputEditText(context, attrs, defStyleAttr) {
```

Comment:
We figured that it was by mistake that `SecurityCodeInput` was extending `CardNumberInput`?

Reply:
Yeah looks like it indeed. Could you remove `open` from `CardNumberInput` as well? So we can't accidentally make the same mistake